mlir/lib/Conversion/EmulateFp8ExtTrunc/EmulateFp8ExtTrunc.cpp

```diff
@@ -365,14 +384,264 @@ static FlatSymbolRefAttr makeFp8TruncFunction(Location loc, FloatType outType,
   return symbolRef;
 }
 
+// Float8E5M2 and Float8E4M3FN
+static FlatSymbolRefAttr makeOCPFp8TruncFunction(Location loc, FloatType outType,
```

Comment:
New function, following the pattern of the other one.  These two types fit the OCP spec, which was handy.

Reply:
I claim that these two functions are separate enough that they might want to be separate functions?

Reply:
... Apparently they're not, never mind